src/System.Device.Gpio/System/Device/Gpio/GpioController.Linux.cs

```diff
@@ -30,6 +31,22 @@ public GpioController(PinNumberingScheme numberingScheme)
         /// <returns>A driver that works with the board the program is executing on.</returns>
         private static GpioDriver GetBestDriverForBoard()
         {
+            try
```

Comment:
We cannot have this logic yet, might be worth adding some utility in your driver for that, i.e.:
`SunxiDriver.GetBestDriver()` or possibly take model info as an input.

Some extensibility model for best driver would be a better approach but we should solve this as separate issue